src/config/dynamic-theme-fixes.config

```diff
@@ -3902,6 +3902,38 @@ INVERT
 
 ================================
 
+jumia.com
+jumia.co
+jumia.ma
+jumia.ci
+jumia.dz
+jumia.ug
+jumia.sn
+zando.co.za
+
+INVERT
+svg.ic[role="img"]:not([aria-label="Zando logo"])
+.logo > a > img
+.inbox > a:not(:nth-of-type(1)) > svg
+img[src*="jumia-group-logo.png"]
+img[src*="jumia_logo_small_checkout.png"]
+img[src*="/Jumia-Pay"]
+li.logo:nth-child(1) > .-i-jumia-logo
+img[src*="empty-cart.png"]
+.-header > img:nth-child(1)
+.-gy5 > .-fs0 > .vent-link[title*="Rewards"]
+.-gy5 > .-fs0 > .vent-link[title*="Pay"]
+.-gy5 > .-fs0 > .vent-link[title*="Food"]
+.-gy5 > .-fs0 > .vent-link[title*="Party"]
+.-gy5 > .-fs0 > .vent-link[title*="Now"]
+nav.s-menu > a.m-itm:nth-child(2) > svg
+nav.s-menu > a.m-itm:nth-child(3) > svg
+nav.s-menu > a.m-itm:nth-child(4) > svg
+nav.s-menu > a.m-itm:nth-child(5) > svg
+div.-fw-w:nth-child(1) > a.-fs0 > img
```

Comment:
Info box in zando.co.za
![Zando home info](https://user-images.githubusercontent.com/14346945/100477009-c841bb00-30ef-11eb-82fa-f4d02cb0e91c.jpg)
![Zando home info (2)](https://user-images.githubusercontent.com/14346945/100477012-c972e800-30ef-11eb-8e09-cdafc730a0f4.jpg)